kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -206,22 +206,19 @@ def create(  # noqa: PLR0913, PLR0912
         if group_in_memory:
             nodes, dependencies = group_memory_nodes(context.catalog, pipeline)
         else:
-            nodes = {node.name: [node] for node in pipeline.nodes}
-
-            dependencies = defaultdict(list)
+            nodes = {}
+            dependencies = {}
+            for node in pipeline.nodes:
+                nodes[node.name] = [node]
+                dependencies[node.name] = []
             for node, parent_nodes in pipeline.node_dependencies.items():
                 for parent in parent_nodes:
                     dependencies[parent.name].append(node.name)
 
-        # Sort both parent and child nodes to make sure it's deterministic
```

Comment:
This sorting was replaced with the original topological order obtained from kedro; it is deterministic. Otherwise `nodes` and `dependencies` have a different order which is confusing.